### PR TITLE
Delete follow-up: metadata and options

### PR DESCRIPTION
**Pull request by 2e2a**
See #2374
-  remove 'deleted field in IMetadata sheet in backend
- remove special 'hidden' field for metadatda sheet in options requests (not needed anymore)

### CODE REVIEW

**Comment by 2e2a** [comment]
@xi, @der-john:
As the hidden/deleted differentiation is obsolete, the flags are removed from the IMetadata sheet in the options request (See deletion.rst). This requires  a frontend adaptation.

**Comment by der-john** [comment]
I'm a little out of my depth here: I can see no occurrences of "deleted" in the frontend anymore. And deleting comments works perfectly fine. But the options request for `hide` seems to return `false` all the time, even for god. Some part of the `Http` module probably needs adaption, but I don't which right now.

File: src/adhocracy_core/docs/deletion.html [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+
+&gt;&gt;&gt; resp = admin.get(&quot;/pool1&quot;)
+&gt;&gt;&gt; resp.status_code
+404
+</pre>
+</div>
+</div>
+</div>
+</body>
+</html>
```
Comment (joka) [inline src/adhocracy_core/docs/deletion.html]: this html file does not belong here

**Comment by joka** [comment]
yeah, less code, backend looks good

**Comment by 2e2a** [comment]
tested frontend adaptation

**Pull request merged**